pygeos/strtree.py

```diff
@@ -93,6 +93,24 @@ def query(self, geometry, predicate=None):
 
         return self._tree.query(geometry, predicate)
 
+    def query_bulk(self, geometries, predicate=None):
+
```

Comment:
Can you add a docstring?

Reply:
Specifically, we need to carefully document the return order. 
The first array corresponds to the passed geometries, the second to the tree geometries. I think this makes sense as default*, but I can image this is certainly not obvious / can be surprising.

\*at least for the typical geopandas sjoin case of a point-in-polygon sjoin, the points are the left dataframe, and the polygons the right dataframe, and then you create a tree for the right dataframe, but by default want to keep the order for the left dataframe

Reply:
Docstring is good! It only has to be adjusted for for the new return value